### PR TITLE
First getdist recipe for conda-forge.

### PR DESCRIPTION
**Pull request by vitenti**
Checklist
- [X] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [X] Source is from official source.
- [X] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [X] If static libraries are linked in, the license of the static library is packaged.
- [X] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [X] Build number is 0.
- [X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [X] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [X] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by vitenti** [comment]
@conda-forge/help-python, ready for review!

File: recipes/getdist/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8
+    - pip
+  run:
+    - python >=3.8
+    - numpy >=1.17.0
+    - matplotlib-base (>=2.2.0,!=3.5.0)
```
Comment (beckermr) [inline recipes/getdist/meta.yaml:27]:
```suggestion
    - matplotlib-base >=2.2.0,!=3.5.0
```

Comment (beckermr) [inline recipes/getdist/meta.yaml:27]:
```suggestion
    - matplotlib-base >=2.2.0,!=3.5.0
```

File: recipes/getdist/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    - scipy >=1.5.0
+    - PyYAML >=5.1
```
Comment (beckermr) [inline recipes/getdist/meta.yaml:29]:
```suggestion
    - pyyaml >=5.1
```

File: recipes/getdist/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    GetDist is a Python package for analysing Monte Carlo samples, including 
+    correlated samples from Markov Chain Monte Carlo (MCMC).
+  license: LGPL-3.0-or-later
+  license_file: LICENSE.txt
+  doc_url: https://getdist.readthedocs.io/
+
+extra:
+  recipe-maintainers:
+    - vitenti
+    - marcpaterno
```
Comment (beckermr) [inline recipes/getdist/meta.yaml:48]: All maintainers on the recipe besides the person who submitted need to comment here that they are ok maintaining the recipe.

Comment (marcpaterno) [reply recipes/getdist/meta.yaml:48]: I am OK maintaining this recipe.

**Comment by beckermr** [comment]
merge conflicts need fixing - then we can merge

**Pull request merged**